services/tests/test_archive.py

```diff
@@ -1,18 +0,0 @@
-from pathlib import Path
-from unittest.mock import patch
-
-from django.test import TestCase
-from shared.django_apps.core.tests.factories import RepositoryFactory
-
-from services.archive import ArchiveService
-
-current_file = Path(__file__)
-
-
-class ArchiveServiceTests(TestCase):
-    @patch("services.storage.StorageService.create_presigned_put")
-    def test_create_raw_upload_presigned_put(self, create_presigned_put_mock):
```

Comment:
Do these tests exist in Shared already?

Reply:
They should, yes.
I extracted the `ArchiveService` move to https://github.com/codecov/codecov-api/pull/1006, as it was indeed confusing to mix these changes.